lib/src/model/movies_result_model.dart

```diff
@@ -0,0 +1,65 @@
+class Result {
```

Comment:
the class should have the same name as the file, to keep consistency

Reply:
Done